### DISCUSSION TITLE
posts/2023/10/reading-meeting/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#26] [General]
# posts/2023/10/reading-meeting/

https://guozheng.rbind.io/posts/2023/10/reading-meeting/

**Comment by earfanfan** [comment]
哇，本社畜也好想参加这样的读书会啊，虽然你写的我没怎么看懂，但我很好奇“使用社会学的视角思考社会热点现象”是什么。

**Reply by residualsun1** [reply]
我们上个学期就开始了！（还是念书好哇）

记得之前益辉写过一篇博文，讲的是要好好写一篇文章挺难的。我感觉自己写博文还是有种没“好好写”的状态，我想这是导致你没看懂的原因。老早我就在心里说要把之前的博文全都“翻修”一遍，不然自己可能都不忍心回头看。关于如何“使用社会学的视角思考社会热点现象”，其实我也不大懂，估计也入门也没到，还在阅读和写作的过程中积累，希望能尽早写出让大家看得懂的内容。

---

另，我发现你最近还读了蛮多本书，而且还坚持写作。比我身边许多念社科的学生还要勤快，阅读量也更丰厚！佩服你，也向你学习。（不自觉想到充满竞争的今天，阅读这件门槛很低的事情却似乎被很多应该以此为自身养料的我以及我身边的同学忽视，倒是我在论坛里遇到的很多理工科出身的小伙伴多少都给我“腹有诗书”而又并不在意，该读书还是读书的状态。）

**Reply by earfanfan** [reply]
可能不是因为你没好好写，我觉得就只是因为你在写这篇博客时用了你习惯的那些专业名词，而我这门外汉确实不懂而已。现在写博客就用自己用习惯的词就好，等到你变成大佬了再追求去把专业的东西写得通俗易懂吧。

阅读这件事门槛其实不低，论坛里那些大佬们大概是已经很厉害了，不特别把读书看做吸取养料，而就是读自己想读的书而已，估计读书对大佬们来说跟写代码、种菜、做饭也差不多，已经不需要刻意追求撒了。我倒是还处于要靠看书来认识世界的阶段，最近看的书都是需要静下心来看的那种，基本上都让我感到震撼。
